app/contributing/word-choice.md

```diff
@@ -60,13 +61,29 @@ Use <i class="fa fa-check"></i> | Don't use <i class="fa fa-times"></i>
 Self-managed                    | Self-hosted, on-premise(s)
 Free tier                       | Free mode
 Plus tier                       | Plus mode, Plus subscription
-Enteprise tier                  | Enterprise mode, Enterprise subscription
+Enterprise tier                  | Enterprise mode, Enterprise subscription
 Cloud                           | Software as a Service/SaaS
 cURL                            | curl, CURL
 HTTPie                          | httpie, HTTPIE
 Unix                            | unix, UNIX
 Nginx                           | nginx, NGINX
 
+
+### List punctuation
+Don't use punctuation when creating ordered and unordered lists that are extensions of a sentence:
```

Comment:
I wonder if we can add a green checkmark here and a red X before "don't" on line 80 to make the do/don't content stand out.